### DISCUSSION TITLE
Config issues

### DISCUSSION THREAD

**Discussion by ariscript** [JanDeDobbeleer/oh-my-posh#1062] [Troubleshoot] [answered]
Currently, I have [this config](https://srcb.in/qz1VptUcPx) set up (intending it to be cross-platform for powershell and zsh in wsl if possible). I have it in a satisfactory state **EXCEPT** for some small issues:

- there is no space after the hourglass icon even if I add a space at the end of the `prefix` in the execution time segment (it gets added in front of it for some reason)
- there are two spaces after the `#` prompt but I only want one.

![image](https://user-images.githubusercontent.com/57048370/137607460-85cbdb0c-9209-42be-a8fd-9a3b3e6cda5b.png)

Also, I would love to find a resource on what unicode character corresponds to what icon as all I can do now is look around in the docs and figuring out what they are based on context.

**Comment by lnu** [comment, marked as answer]
For the first issue, add a blank space like this:
```
        {
          "type": "executiontime",
          "style": "powerline",
          "foreground": "#cad80c",
          "properties": {
            "prefix": "\ufbab ",
            "display_version": true
          }
        }
```
<img width="105" alt="image" src="https://user-images.githubusercontent.com/1829553/137614070-37545cc6-399b-49b2-bfcf-07df316dc0aa.png">

and for the second one ` "final_space": false,`
<img width="34" alt="image" src="https://user-images.githubusercontent.com/1829553/137614101-5b7df663-cf0e-4fb3-98b1-9046677ebc0b.png">

**Reply by JanDeDobbeleer** [reply]
About the question on icons: https://www.nerdfonts.com/cheat-sheet

**Reply by ariscript** [reply]
> 
> 
> For the first issue, add a blank space like this:
> 
> ```
>         {
>           "type": "executiontime",
>           "style": "powerline",
>           "foreground": "#cad80c",
>           "properties": {
>             "prefix": "\ufbab ",
>             "display_version": true
>           }
>         }
> ```
> 
> <img alt="image" width="105" src="https://user-images.githubusercontent.com/1829553/137614070-37545cc6-399b-49b2-bfcf-07df316dc0aa.png">

This somehow results in an extra space being added BEFORE the hourglass icon:
![image](https://user-images.githubusercontent.com/57048370/137630473-9fdbb7b3-0651-44d2-9708-22b271a02854.png)

(the `final_space` works now so thanks for that!)

**Reply by lnu** [reply]
I don't have anything added before the hourglass(see with a space and not at the end):
<img width="131" alt="image" src="https://user-images.githubusercontent.com/1829553/137631832-50aa7aba-2dd4-4696-9f61-03fc4c07c9d6.png">
